Meshtastic/Views/Nodes/Helpers/ShareContactQRDialog.swift

```diff
@@ -61,6 +61,13 @@ struct ShareContactQRDialog: View {
                 .font(.subheadline)
                 .multilineTextAlignment(.center)
                 .foregroundColor(.secondary)
+			ShareLink("Share QR Code & Link",
+						item: Image(uiImage: qrImage),
+					  subject: Text("Import Meshtastic Node \(node.user.shortName) as a contact"),
```

Comment:
I think it would good to remain consistent with terminology and usage of longname vs shortname. 
Add vs Import and shortname vs longname